TagListView.podspec

```diff
@@ -1,12 +1,14 @@
 Pod::Spec.new do |s|
   s.name         = "TagListView"
-  s.version      = "1.3.1"
+  s.version      = "1.3.2"
```

Comment:
we already have 1.3.2 internally. we should bump to 1.3.3

Reply:
Done, thanks!